RealmSwift/Migration.swift

```diff
@@ -127,7 +127,7 @@ public final class Migration {
     - returns: The created object.
     */
     @discardableResult
-    public func createObject(ofType typeName: String, populatedWith value: Any = [:]) -> MigrationObject {
+    public func create(_ typeName: String, value: Any = [:]) -> MigrationObject {
```

Comment:
Still complies with Swift 3 style without being too much of a breaking change for our existing Swift 2.2 users IMO.